### DISCUSSION TITLE
Does fooocus need dedicated graphic card to run ?

### DISCUSSION THREAD

**Discussion by htetlwink** [lllyasviel/Fooocus#3684] [Q&A] [answered]
I want to test the Fooocus but my laptop does not have dedicate graphic card and when i start the "run.bat" file. it download the model and prompt show " 

> RuntimeError: Found no NVIDIA driver on your system. Please check that you have an NVIDIA GPU and installed a driver from http://www.nvidia.com/Download/index.aspx

" is there any way can i run the software without the dedicated graphic card. HELP...

**Comment by Rogala** [comment, marked as answer]
Edit run.bat

`.\python_embeded\python.exe -s Fooocus\entry_with_update.py --always-cpu`
